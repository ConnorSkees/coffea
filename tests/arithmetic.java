public class arithmetic {  
    public void main(String[] args) {
        int x = 2;
        int y = 3;
        
        int z = (x + y) * x - y;
        double a = Math.pow(x, y);
    }
}