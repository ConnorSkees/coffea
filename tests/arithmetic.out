public class arithmetic {
  public void main(String[] arg1) {
    int i2 = 2;
    int i3 = 3;
    int i4 = (((i2 + i3) * i2) - i3);
    double d5 = Math.pow((double) i2, (double) i3);
    return;
  }
}
